docs/samples/v1beta1/pmml/README.md

```diff
@@ -4,6 +4,13 @@ To test the [PMMLServer](http://dmg.org/pmml/pmml_examples/#Iris) server, first
 
 # Predict on a InferenceService using PMMLServer
 
+## Disadvantages
+
+Because the `pmmlserver` based on [Py4J](https://github.com/bartdag/py4j) and that isn't support multi-process mode. So we can't set `spec.predictor.containerConcurrency`.
```

Comment:
Should we enforce --worker =1 for pmml then?

Reply:
It should be and I'll do it later.